[Bug] bulma-ui semantic-release incorrectly triggers on create-bestax breaking changes

## Affected Package(s)

Which package(s) in the monorepo does this bug relate to?

- [x] bulma-ui (`@allxsmith/bestax-bulma`)
- [ ] create-bestax (`create-bestax`)
- [ ] docs (`@allxsmith/bestax-docs`)
- [ ] Other (please specify):

---

## Description

The bulma-ui semantic-release configuration is incorrectly triggering version bumps when create-bestax has breaking changes. This causes the bulma-ui package to attempt a major version bump even though the breaking change is scoped to create-bestax only.

---

## Reproduction Steps

Steps to reproduce the behavior:

1. Create a commit with message: `feat(create-bestax)!: any breaking change`
2. Semantic release runs for bulma-ui
3. bulma-ui attempts to bump to next major version (e.g., 3.0.0)
4. NPM version command fails due to dependency conflicts

---

## Expected Behavior

bulma-ui semantic-release should ignore commits scoped to create-bestax, including breaking changes.

---

## Actual Behavior

bulma-ui semantic-release attempts to bump version when create-bestax has breaking changes, causing CI failures.

---

## Screenshots / Code

Error from CI:
```
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: feat(create-bestax)!: require Node.js 18+ and align with bestax-bulma v2
[semantic-release] › The next release version is 3.0.0
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
```

---

## Environment

- Package version(s): bulma-ui 2.6.2, create-bestax 1.4.2
- React version: 18.3.1
- Browser (if applicable): N/A
- OS: GitHub Actions Ubuntu
- Node version: 20.x
- Other relevant info: Monorepo with independent versioning

---

## Additional Context

The release.config.js for bulma-ui needs to explicitly ignore create-bestax scoped commits to prevent cross-package version bumping.

--- a/bulma-ui/release.config.js
+++ b/bulma-ui/release.config.js
@@ -7,6 +7,10 @@ export default {
       {
         preset: 'angular',
         releaseRules: [
+          // Explicitly ignore create-bestax scoped commits (including breaking changes)
+          { scope: 'create-bestax', release: false },
+          { breaking: true, scope: 'create-bestax', release: false },
+
           // Release only for bulma-ui scoped commits (independent versioning)
           { type: 'feat', scope: 'bulma-ui', release: 'minor' },
           { type: 'fix', scope: 'bulma-ui', release: 'patch' },

